components/intake-form/WorkExperience.tsx

```diff
@@ -210,6 +216,9 @@ export default function WorkExperience({
             </div>
           </SelectableGrid>
         </div>
+        {showSuggestButton ? null : (
```

Comment:
We should have this show _even after a focus has been suggested_. So, it would start showing from the moment someone considers suggesting and continues until they either remove their suggestion or close the input without adding anything.

I played around with this; you could do it by setting the condition to `!showSuggestButton || focusSuggested` and reversing the ternary (i.e. `<ErrorMessage ... /> : null`).

Reply:
Makes sense! can definitely add that update

Reply:
Done with the latest commit